### PR TITLE
Dri m 1002 mvident

### PR DESCRIPTION
**Pull request by DenisRicard**

### CODE REVIEW

**Review by CZwarts — APPROVED** [review]
Looks good.

**Review by jarrettranderson — APPROVED** [review]

File: ipv-core/src/main/java/com/ibm/whc/deid/util/localization/LocalizationManager.java [hunk 1]
```diff
@@ -130,7 +131,46 @@ private LocalizationManager() {
     }
   }
 
-  private synchronized boolean registerResource(Resource resource, ResourceEntry entry) {
+  /**
+   * Register an advanced custom resource
```
Comment (sfc42) [inline ipv-core/src/main/java/com/ibm/whc/deid/util/localization/LocalizationManager.java:135]: I'd suggest changing this comment.

File: whc-shared/src/main/java/com/ibm/whc/deid/shared/localization/Resources.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
```
Comment (sfc42) [inline whc-shared/src/main/java/com/ibm/whc/deid/shared/localization/Resources.java:1]: Use correct copyright for Alvearie.

**Review by sfc42 — CHANGES_REQUESTED** [review]

File: ipv-core/src/main/java/com/ibm/whc/deid/util/localization/LocalizationManager.java (hunk 1 above)
Comment (DenisRicard) [reply ipv-core/src/main/java/com/ibm/whc/deid/util/localization/LocalizationManager.java:135]: Changing to "register a custom resource"

File: whc-shared/src/main/java/com/ibm/whc/deid/shared/localization/Resources.java (hunk 2 above)
Comment (DenisRicard) [reply whc-shared/src/main/java/com/ibm/whc/deid/shared/localization/Resources.java:1]: Good catch. Thanks!

**Review by sfc42 — APPROVED** [review]

**Pull request merged**